R/set-defaults.R

```diff
@@ -0,0 +1,166 @@
+# ENVIRONMENT ----
+# Environment that holds various global variables and settings for headliner,
+# such as the current default headline syntax. It is not exported.
+headliner_global <- new.env(parent = emptyenv())
```

Comment:
I would consider renaming `headliner_global` so that's it's more apparent that's it's an environment and not just a list of values

Reply:
What kind of names would make this more obvious?

Reply:
I think just `headliner_env` is a good starting point, and is concise. `headliner_env_for_defaults` is the most verbose, but also most clear.